ts/features/itwallet/common/components/ItwSkeumorphicCard/ClaimLabel.tsx

```diff
@@ -21,6 +23,7 @@ export const ClaimLabel: FunctionComponent<
   fontSize = 11,
   fontWeight = "Semibold",
   textTransform = "none",
+  valuesHidden,
```

Comment:
It should be singular

```suggestion
  hidden,
```